src/Microsoft.TestPlatform.AdapterUtilities/ManagedNameUtilities/ManagedNameHelper.Reflection.cs

```diff
@@ -502,62 +509,4 @@ private static string GetTypeString(Type type, bool closedType)
         AppendTypeString(builder, type, closedType);
         return builder.ToString();
     }
-
-#if NET20
```

Comment:
This library by-design supports older frameworks too, we want everyone to be able to use it. Please don't remove this support. I know we don't target these, but if people copy-and-paste this code to their projects, we want it to work. (This point can be discussed though, I don't know if we want to do that :))

Reply:
Minimal version supported by MS is net462 so it should be safe to remove code targeting older versions.